packages/ubuntu_bootstrap/test/storage/guided_resize/storage_size_dialog_test.dart

```diff
@@ -54,19 +54,30 @@ void main() {
       maximumSize: toBytes(100, DataUnit.gigabytes),
     );
     await tester.pumpAndSettle();
+    final context = tester.element(find.byType(StorageSizeDialog));
 
     for (final unit in DataUnit.values) {
-      await tester.tap(find.byType(MenuButtonBuilder<DataUnit>));
+      final menuButton = find.byType(MenuButtonBuilder<DataUnit>).last;
+      expect(menuButton, findsOneWidget);
+      await tester.pumpAndSettle();
+      await tester.ensureVisible(menuButton);
+      await tester.pumpAndSettle();
+      // TODO: This is needed to a bug in Flutter 3.16.0 where the focus still
+      // is on the text field within the SpinBox (only in tests).
+      FocusScope.of(context).unfocus();
```

Comment:
What a nightmare it must have been to debug this test.. thanks a lot!

Reply:
It was an incredible pain. :sweat_smile: